### DISCUSSION TITLE
Flicker when using react-native-pdf and react-native-twilio-video-webrtc (and react navigation) [only on android]

### DISCUSSION THREAD

**Discussion by marqroldan** [software-mansion/react-native-screens#1063] [Q&A] [answered]
I have my modals setup as a screen of react-navigation

I think they're set-up fine; translucent background, nice fade animation that I want

Except the PDF view flickers when going to the modal and exiting the modal

And the video streams (local and remote)  also flicker (and this time when it happens I can see what's behind it 

Layer 1 (initial screen) -> Layer 2 (videos; local + remote) -> Layer 3 (modal)
Layer 2 flickers (specifically only the area where the video is rendered) and I see Layer 1 when it happens

Any ideas on what to do next? 🙏

**Comment by tboba** [comment, marked as answer]
Hi @marqroldan, sorry for late response 🙏 
I don't know if I can visualize your problem in my mind. First thing that comes to my mind is - is your problem still reproducible on react-native-screens 3.26.0? If so, could you make a minimal repro and post it in the issues?
We would love to hear more about your scenario!

I'm closing this discussion right now, but don't hesitate to ask for more details if you need to.
Cheers!
